src/rqt_tf_tree/dotcode_tf.py

```diff
@@ -95,6 +95,14 @@ def generate_dotcode(self,
             self.listen_duration = 0
 
             yaml_data = tf2_frame_srv.call(FrameGraph.Request()).frame_yaml
+
+            def my_construct_mapping(self, node, deep=False):
```

Comment:
```suggestion
            def default_tftree_construct_mapping(self, node, deep=False):
```
I guess this makes the symbol longer than it should be but better than "mine"?